db/migrate/20211116161556_create_reservations.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :reservations do |t|
       t.date :date
       t.string :city
-      t.references :user, null: false, foreign_key: true
-      t.references :car, null: false, foreign_key: true
```

Comment:
The same thing here now is not mandatory to validate that is not null, you're doing this at the model layer, not in the database, this strictly is the same thing, but is not the conventional

Reply:
I removed the type of data that it would be receiving, since I removed the relation from the model, it was creating issues when doing the post request to the api unfortunately, I would probably need to change both the car and the user models in order to make the relationship work